devtools-ci-index.yaml

```diff
@@ -829,6 +829,69 @@
             $ssh_cmd -t "cd payload-tests/integration-tests && /bin/bash cico_run_tests.sh"
     <<: *job_template_defaults
 
+- job-template:
```

Comment:
You're introducing a new template here. Why not just rename the old one?

Reply:
well, it is not used anywhere, we can reuse the old one